app/domain/authentication/authn_jwt/validate_restrictions_one_to_one.rb

```diff
@@ -0,0 +1,19 @@
+module Authentication
+  module AuthnJwt
+
+    # This class is responsible for retrieving the correct value from the JWT token
+    # of the requested attribute.
+    class ValidateRestrictionsOneToOne
```

Comment:
This class usage is not part of this PR.
It makes this PR very hard to read, since reviewer cannot see how do you use this class.
When a new PR will arrive that uses this code, the reviewer will not have the full context to review its correct usage or comment on the original class, since it is not in that PR.

Reply:
Yes you are correct.
Just wanted to try to make my pr's smaller then the one with the get identity

Reply:
that's good effort, just keep them grouped with some meaning 😄